### DISCUSSION TITLE
Transcribing from YouTube

### DISCUSSION THREAD

**Discussion by ManuelMaccou** [deepgram/community#389] [General help] [answered]
I just discovered the YouTube demo [here](https://github.com/deepgram-devs/youtube-demo). I just want to confirm whether or not this increases the cost for each transcription. Does transcribing a 60 minute YouTube video from a link cost the same as uploading a 60 minute audio file?

**Comment by lukeocodes** [comment, marked as answer]
That demo uses `ytdl` to fetch an audio file, and then sends that to Deepgram for transcription! 

Because it sends just plain audio, and only looks like we're transcribing a YouTube video, it is subject to the same costs as transcribing any other audio.

**Reply by ManuelMaccou** [reply]
Sorry I asked two questions which made it confusing. To clarify, this does increase the cost or it does not?

**Reply by lukeocodes** [reply]
No, because it sends just plain audio. It only looks like we're transcribing a YouTube video. So it's subject to the same costs as transcribing audio from anywhere. Hope this helps

(answer updated for clarity)

**Reply by ManuelMaccou** [reply]
Perfect thank you
